core/translate.c

```diff
@@ -1632,7 +1666,10 @@ stress_test_recreate_state(dcontext_t *dcontext, fragment_t *f, instrlist_t *ili
             (!instr_is_our_mangling(in) ||
              /* handle adjacent mangle regions */
              (TEST(FRAG_IS_TRACE, f->flags) /* we have translation only for traces */ &&
-              mangle_translation != instr_get_translation(in)))) {
+              IF_X86((prev_in &&
```

Comment:
What about the comment at line 1663

Reply:
I wasn't sure what it meant. Sounds like something is unsupported for mangle_rel_addr?

Reply:
Looking at the code below, it only checks xcx, xsp, and pc: it's targeting ind branches.

Reply:
Ok, I guess what I've added doesn't have any effect right now, but it should still be correct, leave it? I resolve this, re-open if necessary.